src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -1430,6 +1437,11 @@ public void complete() throws IllegalStateException, IOException, InterruptedExc
                                 } catch (ANTLRException x) {
```

Comment:
Should we keep `project.addTrigger(new PeriodicFolderTrigger("1d"));` or kill it off?

Reply:
Eeeeeh. I'm always a bit squemish about removing behaviour.
That said I didn't know this was a thing :)

Reply:
Yep, there are load of people hacking configuration of this because they don't like it as 1 day... with this change we fix that... but that changes the behaviour for people...

another option would be to check for the property and only set if the property is missing